src/components/AboutSideBar.vue

```diff
@@ -0,0 +1,117 @@
+<template>
```

Comment:
Some self-promotion can't hurt us; Goal is to make it easy for plausible employers to view our GitHub profiles.

Reply:
Let get this bread!